src/constants/EpidemicModel.tsx

```diff
@@ -0,0 +1,5 @@
+export enum RateOfSpreadType {
+  "Controlled",
+  "Infectious",
+  "Missing",
+}
```

Comment:
mentioned elsewhere but this will create a numeric enum, not a string enum with the same interface as what this type looked like before. not sure this is necessarily what you wanted here

Reply:
Misunderstood that. Fixed.